### PR TITLE
NodejsWeek2

### PR DESCRIPTION
**Pull request by dmytriigithub**

### CODE REVIEW

File: nodejs/week2/.gitignore [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+.vscode/settings.json
+
+# React build
+build
+
+dist/
+
+.DS_Store
+
+package-lock.json
```
Comment (Jane-API-SG) [inline nodejs/week2/.gitignore]: It is a good idea to commit the package-lock.json. Using this, you can make sure that you have a service with a working dependency tree; https://docs.npmjs.com/cli/v9/configuring-npm/package-lock-json?v=true. 
It should be removed from this file.

File: nodejs/week2/app.js [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+        res.send(searchResult)
+        return;
+      }
+      res.status(404)
+      throw new Error('Not Found')
+    }
+    res.send(document);
+  }
+  catch (error) {
+    throw error;
```
Comment (Jane-API-SG) [inline nodejs/week2/app.js]: I am not sure it makes sense to catch an error and then throwing it immediately after.
You could consider logging it to the console.

**Review by Jane-API-SG — CHANGES_REQUESTED** [review]
Hi Dmytrii,
I have left you a few comments. Please let me know if anything needs more explaining.

Best Jane

File: nodejs/week2/app.js (hunk 2 above)
Comment (dmytriigithub) [reply nodejs/week2/app.js]: Thanks for review, I have made some changes.

**Review by itsjrm — APPROVED** [review]
Those are some solid updates :ok_hand:

**Pull request merged**